cli/src/diff_util.rs

```diff
@@ -303,7 +306,8 @@ impl<'a> DiffRenderer<'a> {
                     )?;
                 }
                 DiffFormat::NameOnly => {
-                    let tree_diff = from_tree.diff_stream(to_tree, matcher, copy_records);
+                    let tree_diff: BoxStream<CopiesTreeDiffEntry> =
```

Comment:
do you need the type here?

Reply:
no, and i don't know why i added it. removed now.